lib/search-summary/header/index.js

```diff
@@ -1,9 +1,10 @@
 import React, { Component, PropTypes } from 'react';
 import './style.css';
-const postImage = require('../../../src/assets/goingplaces.svg');
+var Scroll = require('react-scroll');
+var scroll = Scroll.animateScroll;
```

Comment:
I think Nikki mentioned a reason for doing it that way but I can't remember. I can change it